reduct-nepal/seep
# All changes

**Tasks Completed

- Added placeholder in text box
- Added legend in diff view of both practice test and actual test
- Removed diff button and any extra elements that are not required
- Added "Take another practice test" button in diff view of practice test

** To Test

- Login as superUser in seep
- route to page /practiceresult page and see if the take another practice test button take you practicetest page or not
- check if the legend is there in the practiceResult page
- check if the diff button is removed from the PracticeResult page
- check if the placeholder is added in the textbox
